.travis.yml

```diff
@@ -18,6 +18,7 @@ script:
       sleep 1
   done
 - source .test-env
+- "curl --header \"X-VAULT-TOKEN: ${VAULT_TOKEN}\" --header \"Content-Type: application/json\" --request POST --data '{\"type\": \"consul\"}'  ${VAULT_ADDR}/v1/sys/mounts/consul"
```

Comment:
I think this might not be a reliable way to mount the backend for testing. I notice that the other tests for the other providers don't need it so I'm thinking there's a different way. Are the backends getting mounted through the `testProviders` in the `Providers` attribute of the test framework? And then torn down via `CheckDestroy`?